scripts/reference_exported_symbols.txt

```diff
@@ -947,6 +947,7 @@ proj_operation_factory_context_set_desired_accuracy
 proj_operation_factory_context_set_grid_availability_use
 proj_operation_factory_context_set_spatial_criterion
 proj_operation_factory_context_set_use_proj_alternative_grid_names
+proj_operation_factory_context_set_discard_superseded
```

Comment:
symbols should be sorted. so move this line just after proj_operation_factory_context_set_desired_accuracy